sr_unix_gnp/cmi_init.c

```diff
@@ -39,79 +39,76 @@ cmi_status_t cmi_init(cmi_descriptor *tnd, unsigned char tnr,
 		size_t usr_size,
 		size_t mbl)
 {
-	cmi_status_t status = SS_NORMAL;
-	char *envvar;
+	cmi_status_t	status = SS_NORMAL;
+	char		*envvar;
 	struct protoent *p;
-	unsigned short myport;
-	sigset_t oset;
-	int on = 1;
-	int rval, rc, save_errno;
+	unsigned short	myport;
+	sigset_t	oset;
+	int		on = 1;
+	int		rval, rc, save_errno;
 	struct addrinfo	*ai_ptr, *local_ai_ptr;
 	boolean_t	af;
 
 	ASSERT_IS_LIBCMISOCKETTCP;
 	status = cmj_netinit();
 	if (CMI_ERROR(status))
 		return status;
-
 	status = cmj_getsockaddr(NULL, tnd, &local_ai_ptr);
 	if (CMI_ERROR(status))
 		return status;
 	ntd_root->pool_size = pool_size;
 	ntd_root->usr_size = usr_size;
 	ntd_root->mbl = mbl;
-
 	p = getprotobyname(GTCM_SERVER_PROTOCOL);
 	endprotoent();
 	if (!p)
+	{
```

Comment:
Before I forget, I think we need to mention Coverity scan in the commit comment first line as that is what identified all these issues.